src/website/src/components/layouts/TopBanner.tsx

```
@@ -79,6 +82,33 @@ const TopBanner = () => {
+      normalizedRequested.split('-')[0] === DEFAULT_GOOGLE_LANGUAGE;
+
+    if (isDefaultLanguage) return;
+
+    const retryTimeouts = [120, 800, 1800].map((delay) =>
+      window.setTimeout(() => {
+        void applyGoogleTranslateLanguage(requestedLanguage, 1800);
+      }, delay),
+    );
+
+    return () => {
+      retryTimeouts.forEach((timeoutId) => window.clearTimeout(timeoutId));
```

Biome (2.4.6)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)